### PR TITLE
Azure sample update

### PR DESCRIPTION
**Pull request by plooploops**
Updated azure sample for terratest (using go 1.14) and azure-sdk-for-go version.

cc: @jsturtevant

### CODE REVIEW

File: examples/terraform-azure-example/README.md [hunk 1]
```diff
@@ -41,5 +41,90 @@ it should be free, but you are completely responsible for all Azure charges.
...
+1. Make sure [the azure-sdk-for-go versions match](#check-go-dependencies) in the go.mod file and in the test Go file.
+1. [Check environment variables](#check-environment-variables).
 1. `go test -v -run TestTerraformAzureExample`
+
+
+
+
+## Check Go Dependencies
+
+Check that the /azure-sdk-for-go version must match the version in the terratest mod.go file.
```
Comment (jsturtevant) [inline examples/terraform-azure-example/README.md]:
```suggestion
Check that the `github.com/Azure/azure-sdk-for-go` version must match the version in the terratest [go.mod](https://github.com/gruntwork-io/terratest/blob/master/go.mod) file.  
```

File: examples/terraform-azure-example/README.md [hunk 2]
```diff
@@ -41,5 +41,90 @@ it should be free, but you are completely responsible for all Azure charges.
...
-1. `dep ensure`
+1. `go mod init github.com/<yourrepo>/terratest-module`
+1. `go build terraform_azure_example_test.go`
+1. Make sure [the azure-sdk-for-go versions match](#check-go-dependencies) in the go.mod file and in the test Go file.
+1. [Check environment variables](#check-environment-variables).
 1. `go test -v -run TestTerraformAzureExample`
+
+
+
+
```
Comment (jsturtevant) [inline examples/terraform-azure-example/README.md]: There is extra white space here

File: examples/terraform-azure-example/README.md [hunk 3]
```diff
@@ -41,5 +41,90 @@ it should be free, but you are completely responsible for all Azure charges.
...
+## Check Go Dependencies
+
+Check that the /azure-sdk-for-go version must match the version in the terratest mod.go file.  
+
+> This was tested with **go1.14.1**.  We have included a sample **go.mod** to correspond with the terraform_azure_Example_test.go test, but these steps will include details on how to generate the go module and include matching dependencies.
```
Comment (jsturtevant) [inline examples/terraform-azure-example/README.md]: I think adding a sample go.mod file will be confusing long term as the go.mod file will change frequently over time and this sample is likely not going to stay up to date.

Comment (plooploops) [reply examples/terraform-azure-example/README.md]: Sure, will point to the existing one then.

File: examples/terraform-azure-example/README.md (hunk 1 above)
Comment (plooploops) [reply examples/terraform-azure-example/README.md]: Added in the suggestion.

File: examples/terraform-azure-example/README.md (hunk 2 above)
Comment (plooploops) [reply examples/terraform-azure-example/README.md]: Removed extra whitespace.

**Comment by yorinasub17** [comment]
Going to merge this in. Thanks for your contribution!

**Pull request merged**